server/storage.go

```diff
@@ -43,34 +43,39 @@ func flushMapToTire2Storage() error {
 func (s *Server) sendToTier2Storage(storageType interface{}, buf []byte, seq uint64, tierStorageType string) error {
 	storedType := reflect.TypeOf(storageType).Elem().Name()
 	var streamName string
+	var tieredStorageEnabled bool
```

Comment:
please move this check to the filestore/memstore file where you making the check for $memphis prefix